tools/generator/src/Generator+AddBazelDependenciesTarget.swift

```diff
@@ -184,6 +188,9 @@ if [ "$ACTION" == "indexbuild" ]; then
   # We use a different output base for Index Build to prevent normal builds and
   # indexing waiting on bazel locks from the other
   output_base="$OBJROOT/bazel_output_base"
+  convenience_symlinks=ignore
+else
+  convenience_symlinks=normal
```

Comment:
Is this being used somewhere? Is it supposed to be passed to `--experimental_convenience_symlinks`?

Reply:
Accidental commit from when I was trying something else. I'll remove.